NOTICE.md

```diff
@@ -19,6 +19,10 @@ Thanks goes to these wonderful people (sorted by last name):
  * Daniel Buehler
  * Sebastian Fischer
  * Damien Kayser
+ * Sebastian Clerson
```

Comment:
I'm a bit confused about the visualization here. Why are these elements in green and look like they are not there yet. When I check https://raw.githubusercontent.com/eclipse/kiso-testing/master/NOTICE.md, there are already there

Reply:
I take the last master, but I think I have oti take the branch origin/feature/one-year-contrbution ?

Reply:
Maybe it comes from the pre-commit hook and trailing white spaces